lib/icinga/apiactions.cpp

```diff
@@ -504,11 +510,16 @@ Dictionary::Ptr ApiActions::RemoveDowntime(const ConfigObject::Ptr& object,
 		downtime->SetRemovedBy(author);
 	}
 
-	String downtimeName = downtime->GetName();
+	try {
+		String downtimeName = downtime->GetName();
+		Downtime::RemoveDowntime(downtimeName, true);
 
```

Comment:
What's this for?

Reply:
what do you mean, "what's this for?" ? If you want to know what the new variable is for then I need it to have an API response with the appropriate downtime name in case the downtime is successfully deleted. I can't access the downtime name afterwards once it is deleted.

Reply:
Sorry, I meant the removal of the newline separating `String downtimeName = downtime->GetName();` and `Downtime::RemoveDowntime(downtimeName, true);`.